admin/Default/newsletter_send_processing.php

```diff
@@ -11,7 +11,7 @@
 
 //$mail->ConfirmReadingTo       = 'newsletter-read@smrealms.de';
 
-$mail->AddReplyTo('newsletter@smrealms.de', 'SMR Support');
+$mail->AddReplyTo('support@smrealms.de', 'SMR Support');
```

Comment:
This line should be unnecessary right? Because it's always overridden below?

Reply:
When you send a newsletter to a single recipient, it does not get overwritten.

Reply:
Ah ok, it might be worth moving it into that branch then to make the distinction clear

Reply:
Good idea. I've made the change in PR #340